### PR TITLE
WMS-ID: #11353

### PR DESCRIPTION
**Pull request by flynnkc**
# Add here your WMS ID: 11353

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by KillianLynch — COMMENTED** [review]
@flynnkc, @anooshapilli  - Hi Anoosha, is the format for oci labs( numbering on lab names & the folder named 'workshop/tenancy') OK?

File: oci-cis-landing-zone/04-apply-inspect/apply.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- Create a Terraform Plan using OCI Resource Manager
+- Inspect the Plan for changes
+- Use the Plan to Terraform Apply
+- Inspect the Apply output
+
+### Prerequisites
+
+Previous Labs
+
+### Concepts
```
Comment (anooshapilli) [inline oci-cis-landing-zone/04-apply-inspect/apply.md:22]: You can rename it to About Plan and Apply, make it heading 3 ### and move it before the objectives section.

File: oci-cis-landing-zone/06-compliance/compliance.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### Objectives
+
+In this lab, you will:
+
+- Use the Cloud Shell to run the Compliance Checker script
+- Download the script output to your local machine
+- Inspect the report findings
+
+### Prerequisites
```
Comment (anooshapilli) [inline oci-cis-landing-zone/06-compliance/compliance.md:17]: If there are no prerequisites, you can remove the section. it's optional.

**Review by anooshapilli — COMMENTED** [review]
@flynnkc - 

1. Please remove the numbers before the folder names and update the manifest.json 

2. Please use dashes between words in titles instead of space or underscore.
 
image

3. Please update these markdown file names to match the folder names like the other labs.

image

**Comment by flynnkc** [comment]
@anooshapilli I've incorporated the changes you specified earlier. Please review again at your earliest convenience.

Thanks,
KC

**Pull request closed without merging**